packages/snjs/lib/services/item_manager.ts

```diff
@@ -742,7 +742,7 @@ export class ItemManager extends PureService {
                 a.title,
                 b.title
               )
-          : (a, b) => a.title.localeCompare(b.title)
+          : (a, b) => a.title.localeCompare(b.title, 'en', { numeric: true })
```

Comment:
Nice. We wanted to add this as well for sorting by title in the notes list. Can be done in a separate PR though.